src/components/common/Footer.tsx

```diff
@@ -0,0 +1,79 @@
+import React from "react";
+import "./Footer.css";
+import fw from "@assets/svg/FW200.svg";
+import instagram from "../../assets/images/instagram_logo.png";
+
+const Footer = () => {
+	return (
+		<div className="footer">
+			<div className="footer section__padding">
+				<div className="footer-links">
+					<div className="footer-links-div">
+						<div className="logo">
+							<img src={fw} alt="fw" className="logo-image" />
+						</div>
+					</div>
+					<div className="footer-links-div">
+						<a href="/contribute">
```

Comment:
Just a comment, as we're using React Router Dom, it's better to use the "useNavigate()" hook, but href is fine as well in this case. An example of a use case here would be:

```ts
//before return()
const navigate = useNavigate();

//this section/in the return() body

<div
className="footer-links-div"
>
  <Typography
    variant="body1"
    onClick={() => navigate("/contribute")}
  >
      Contribute
  </Typography>
</div>
```